docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -1232,21 +1232,21 @@ Default: `1GB`
 
 The Admin UI for the YB-TServer is available at `http://localhost:9000`.
 
-### Home
+### Dashboards
 
-Home page of the YB-TServer (`yb-tserver`) that gives a high level overview of this specific instance.
+List of all dashboards to review ongoing operations.
 
-![tserver-home](/images/admin/tserver-home.png)
+![tserver-dashboards](/images/admin/tserver-dashboards.png)
 
-### Dashboards
+### Tables
 
-Here's a list of all dashboards to review the ongoing operations:
+List of all tables managed by this specific instance, sorted by namespace.
```

Comment:
Add Link to namespaces here : https://deploy-preview-18410--infallible-bardeen-164bc9.netlify.app/preview/reference/configuration/yb-master/#namespaces